### PR TITLE
Haoyuanli update readme

### PR DESCRIPTION
**Pull request by haoyuanli**
the screenshots will be available once the PR is merged

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,7 +21,7 @@ git clone git@github.com:CanDIG/candigv2_dashboard.git
 cd candigv2_dashboard
```
Comment (fcoralsasso) [inline README.md:21]: Shouldn't be `cancogen_dashboard`?

Comment (haoyuanli) [reply README.md]: fixed

File: README.md [hunk 2]
```diff
@@ -30,18 +32,18 @@ npm start
...
 
 ![](https://raw.githubusercontent.com/CanDIG/candigv2_dashboard/develop/docs/datasets_dropdown.png)
 
 ### Available pages
 
-Below there is a list of some (but not limited to) available pages for data visualization. Click on the picture to expand it.
+Below there is a list of screenshots of selected pages. Click on the picture to expand it.
 
-| ![](https://raw.githubusercontent.com/CanDIG/candigv2_dashboard/develop/docs/overview_page.png)        | ![](https://raw.githubusercontent.com/CanDIG/candigv2_dashboard/develop/docs/patients_overview.png) |
+| ![](https://raw.githubusercontent.com/CanDIG/candigv2_dashboard/develop/docs/overview_page.png)        | ![](https://raw.githubusercontent.com/CanDIG/candigv2_dashboard/develop/docs/htsget_browser.png) |
```
Comment (fcoralsasso) [inline README.md:43]: We should fix all of these urls from `candigv2_dashboard` to `cancogen_dashboard`

**Review by fcoralsasso — APPROVED** [review]
lgtm

**Pull request merged**